astronomer/airflow/version_check/update_checks.py

```diff
@@ -249,9 +249,15 @@ def parse_version(rel):
                 else pendulum.now('UTC')
             )
 
-            end_of_support = (
-                pendulum.parse(release.get('end_of_support'), timezone='UTC')
-                if release.get('end_of_support')
+            end_of_maintenance = (
+                pendulum.parse(release.get('end_of_maintenance'), timezone='UTC')
+                if release.get('end_of_maintenance')
+                else None
```

Comment:
Doesn't have to be this PR, but we should start to require these new fields in the schema.